python/wheel_install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Use `-` (`wheel-install.sh`), unless there's a reason not to.

Reply:
Done